MapleServer2/PacketHandlers/Game/GuildHandler.cs

```diff
@@ -61,19 +61,26 @@ private void HandleCreate(GameSession session, PacketReader packet)
         {
             string guildName = packet.ReadUnicodeString();
 
-            session.Player.Mesos -= 2000;
+            if (session.Player.Mesos >= 2000)
```

Comment:
can you instead invert this check, then put the `TODO: reject packets` part here with the return? then the code after that doesn't need to live inside the `else` block